test/PerformanceTests/Benchmarks/Sequence/HttpTriggers.cs

```diff
@@ -4,16 +4,13 @@
 namespace PerformanceTests.Sequence
 {
     using System;
-    using System.IO;
     using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Http;
     using Microsoft.AspNetCore.Mvc;
     using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.DurableTask;
     using Microsoft.Azure.WebJobs.Extensions.Http;
-    using Microsoft.AspNetCore.Http;
     using Microsoft.Extensions.Logging;
-    using Newtonsoft.Json;
-    using Microsoft.Azure.WebJobs.Extensions.DurableTask;
-    using Microsoft.WindowsAzure.Storage.Queue;
```

Comment:
These changes primarily about removing and sorting using, as Microsoft.WindowsAzure.Storage.Queue is no longer used with WebJobs.Extensions.DurableTask v3.